File: web3rpc/rpc-specs/code-samples/python/debug/logging/verbosityByName.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

name = "API"
level = 1

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.verbosity_by_name(name, level)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/verbosityByName.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const name = "API";
  const level = 1;

  provider.debug
    .verbosityByName(name, level, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
